### PR TITLE
Two new samples + docs refactoring

### PR DESCRIPTION
**Pull request by omri374**
## Change Description

Added two samples:
1. Batch processing (sample uses Pandas DataFrames)
2. Getting the text using a custom Operator

Updated documentation:
1. Added section on replacing the NLP model
2. Bug fixes
3. Moved the menu to the left as it gets hidden (pinning it is not supported by the default mkdocs-material package, only via the Insiders version)

## Issue reference

This PR provides examples for issues #742, #600, #609

## Checklist

- [X] I have reviewed the [contribution guidelines](https://github.com/microsoft/presidio/blob/main/CONTRIBUTING.md)
- [X] I have signed the CLA
- [ ] My code includes unit tests
- [ ] All unit tests and lint checks pass locally
- [X] My PR contains documentation updates / additions if required

### CODE REVIEW

File: docs/samples/python/getting_entity_values.ipynb [hunk 1]
```diff
@@ -0,0 +1,171 @@
+{
```
Comment (navalev) [inline docs/samples/python/getting_entity_values.ipynb:1]: Nice usage of the lambda operator for this!

**Review by navalev — APPROVED** [review]
looks great! run the 2 notebooks and all good :)

File: mkdocs.yml [hunk 2]
```diff
@@ -62,7 +63,8 @@ theme:
   favicon: assets/ms_icon.png
   features:
     - navigation.instant
-    - navigation.tabs
+    # - navigation.tabs
```
Comment (SharonHart) [inline mkdocs.yml:66]: Intentional?

**Review by SharonHart — APPROVED** [review]
Looks great! adds lots of great functionality. I don't see why not to merge it to the source code...

File: mkdocs.yml (hunk 2 above)
Comment (omri374) [reply mkdocs.yml:66]: Yes, the tabs above are hidden if you scroll. The only way to get them to stay is to upgrade mkdocs-material to the Insiders version.

**Pull request merged**